packages/liquid-html-parser/grammar/liquid-html.ohm

```diff
@@ -409,7 +409,7 @@ LiquidDoc <: Helpers {
   // We use this as an escape hatch to stop matching TextNode and try again when one of these characters is encountered
   openControl:= "@" | end
 
-  descriptionNode = "@description" strictSpace* descriptionContent
+  descriptionNode = "@description" space* descriptionContent
```

Comment:
Now, we are matching `newlines` as well. This allows us to check for a newline character in index `[1]`.

If it is **not** present, then we know that the description content was provided in line.